Manifest.json

```diff
@@ -91,7 +91,7 @@
   },
   "requires": {
     "@qooxdoo/compiler": "^1.0.0-beta",
-    "@qooxdoo/framework": "6.0.0-alpha"
+    "@qooxdoo/framework": "6.0.0-beta"
```

Comment:
Does the framework require itself here?

Reply:
Not the framework. But the schema requires this. And the Compiler requires a Manifest.json. Without you get an error.

Reply:
Ok, understood. But it should probably be `"@qooxdoo/framework": "^6.0.0-beta"`  (vith ^), so that you don't have to update it when we release 6.0.0...

Reply:
Version will be set to current version during deployment. See tool/admin/bin/bumpqxversion.py

Reply:
Duh. Of course! 😄